packages/wrangler/src/__tests__/pipelines.test.ts

```diff
@@ -581,23 +581,23 @@ describe("pipelines", () => {
 				{
 					type: "http",
 					format: "json",
-					config: {
+					configuration: {
```

Comment:
should this not be just `authenticated: true`?

Reply:
oops.  this change was left in my workspace.  have added and rebased.